### PR TITLE
Search customer by external_id (Import API)

### PR DESCRIPTION
**Pull request by promulo**
I've included a very simple `.find_by_external_id` method in `Import::Customer`.

This would avoid the following pattern for retrieving one particular customer:
```
customer = ChartMogul::Import::Customer.all(external_id:"cus_0001").first
# do something with customer object (e.g. cancel subscription)
```

### CODE REVIEW

File: spec/chartmogul/import/customer_spec.rb [hunk 1]
```diff
@@ -86,6 +86,23 @@
     end
   end
 
+  describe '.find_by_external_id', vcr: true do
+    let(:customer) { ChartMogul::Import::Customer.create!(attrs) }
```
Comment (jasonl) [inline spec/chartmogul/import/customer_spec.rb]: There's no need to create the customer here - VCR will replay the request, so you can simply fetch an existing customer.

Remember to regenerate the API keys in your test ChartMogul account when you do this, as they're accessible in the .yml fixtures.

Comment (promulo) [inline spec/chartmogul/import/customer_spec.rb]: Thanks for checking, I've fixed that.

**Comment by jasonl** [comment]
:+1:

**Comment by DmitryDrobotov** [comment]
👍

**Pull request merged**